rasa/core/tracker_store.py

```diff
@@ -56,11 +56,28 @@ class TrackerStore:
     """Class to hold all of the TrackerStore classes"""
 
     def __init__(
-        self, domain: Optional[Domain], event_broker: Optional[EventBroker] = None
+        self,
+        domain: Optional[Domain],
+        event_broker: Optional[EventBroker] = None,
+        load_events_from_previous_conversation_sessions: bool = False,
```

Comment:
Optional: call this `retrieve_events_from_previous_conversation_sessions` to match the underlying call to `retrieve()` that's affected